### ISSUE TITLE
(Feature request) Port-specific rules

### ISSUE THREAD

**Issue by spectreseven1138** [MERLev/reVita#2]
Hi, I think it would be great to have a way to create rules that apply only to for a specific controller port, so that you can have mappings for different controllers or players. Would be useful in games like Minecraft where the ideal trigger and dpad mappings are different for the Vita and the DS4. Or on a local multiplayer game where two players have different control preferences

Not 100% sure if this is possible (I think it should be since the inputs of each port can be seen in the developer menu), but if it is then this would be a great feature to have. Either way, thanks for your work on this great plugin.

**Comment by MERLev** [comment]
Yeah, it is totally possible - but is not planned in near future.
